### PR TITLE
Changed Formatting of D Game Engines.

### PR DESCRIPTION
**Pull request by d-static**
Changed Formatting of D Game Engines. Added a subsection for bindings, and a subsection for pure D game engines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,9 +433,12 @@
 	* OpenAL (DerelictAL), 
 	* ALURE (DerelictALURE).
 
-## Game Engine Bindings
+## Game Engine
```
Comment (maxhaton) [inline README.md:436]:
```suggestion
## Game Engines
```

File: README.md [hunk 2]
```diff
@@ -433,9 +433,12 @@
...
 
-* [Godot-D](https://github.com/godot-d/godot-d) - D language bindings for the Godot Engine's GDNative API
+* Game Engine Bindings:
```
Comment (maxhaton) [inline README.md:438]:
```suggestion
### Game Engine Bindings:
```

?

File: README.md [hunk 3]
```diff
@@ -48,8 +48,8 @@
 	- OS
 		- [Operating Systems](#os)
 	- Gaming
-		- [Bindings](#game-bindings)
-		- [Game Engine Bindings](#game-engine-bindings)
+		- [Game and Graphics Bindings](#game-bindings)
```
Comment (WebFreak001) [inline README.md:51]: You need to also change the header in the content if you change the ToC

If you change the header, add an `<a name...` tag like occuring elsewhere in the document, to give backwards compatibility for old links.

File: README.md [hunk 4]
```diff
@@ -433,9 +433,12 @@
...
 	* ALURE (DerelictALURE).
 
-## Game Engine Bindings
+## Game Engine
 
-* [Godot-D](https://github.com/godot-d/godot-d) - D language bindings for the Godot Engine's GDNative API
+* Game Engine Bindings:
+	* [Godot-D](https://github.com/godot-d/godot-d) - D language bindings for the Godot Engine's GDNative API
+
+* Pure D Game Engines:
```
Comment (WebFreak001) [inline README.md:441]:
```suggestion
### D Game Engines
```

File: README.md (hunk 2 above)
Comment (WebFreak001) [reply README.md:438]:
```suggestion
### Game Engine Bindings
```

**Review by WebFreak001 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (WebFreak001) [reply README.md]: not resolved

File: README.md (hunk 2 above)
Comment (WebFreak001) [reply README.md]: not resolved

File: README.md (hunk 4 above)
Comment (WebFreak001) [reply README.md:441]: not resolved

**Review by WebFreak001 — CHANGES_REQUESTED** [review]
please don't just dismiss our reviews and actually change what we ask you to, or at least explain why you dismiss our changes

File: README.md (hunk 3 above)
Comment (WebFreak001) [reply README.md:51]: still needs change

File: README.md [hunk 5]
```diff
@@ -433,9 +433,12 @@
 	* OpenAL (DerelictAL), 
 	* ALURE (DerelictALURE).
 
-## Game Engine Bindings
+## Game Engines
 
-* [Godot-D](https://github.com/godot-d/godot-d) - D language bindings for the Godot Engine's GDNative API
+### Game Engine Bindings:
```
Comment (WebFreak001) [inline README.md:438]:
```suggestion
### Game Engine Bindings
```

File: README.md [hunk 6]
```diff
@@ -48,8 +48,8 @@
 	- OS
 		- [Operating Systems](#os)
 	- Gaming
-		- [Bindings](#game-bindings)
-		- [Game Engine Bindings](#game-engine-bindings)
+		- [Game and Graphics Bindings](#game-bindings)
+		- [Game Engines](#game-engines)
```
Comment (WebFreak001) [inline README.md:52]: I think you meant to link to `D Game Engines` here? It's below Game and Graphics Bindings at least

Comment (d-static) [reply README.md:52]: RIght

**Pull request closed without merging**